src/coreclr/debug/runtimeinfo/datadescriptor.h

```diff
@@ -152,6 +152,18 @@ CDAC_TYPE_FIELD(PreviousNestedInfo, /*pointer*/, PreviousNestedInfo, offsetof(Ex
 #endif
 CDAC_TYPE_END(ExceptionInfo)
 
+CDAC_TYPE_BEGIN(ExceptionObject)
+CDAC_TYPE_INDETERMINATE(ExceptionObject)
+CDAC_TYPE_FIELD(ExceptionObject, /*pointer*/, Message, cdac_offsets<ExceptionObject>::Message)
```

Comment:
I would always use the exact managed type field names in the data contract descriptor so that we do not need to worry about having different names for the same thing in different places.

The managed type field names are often set in stone because of binary serialization or because of existing (unwritten) diagnostic contract.